app/src/test/java/com/github/se/cyrcle/model/user/UserViewModelTest.kt

```diff
@@ -0,0 +1,90 @@
+package com.github.se.cyrcle.model.user
+
+import kotlinx.coroutines.runBlocking
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.verify
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class UserViewModelTest {
+
+  @Mock private lateinit var userViewModel: UserViewModel
```

Comment:
The user viewModel that is tested should not be a mock. The annotation is not needed

Reply:
Indeed